srcjs/output_binding_image.js

```diff
@@ -225,6 +225,18 @@ outputBindings.register(imageOutputBinding, 'shiny.imageOutput');
 
 var imageutils = {};
 
+imageutils.disableDrag = function($img) {
+  // Make image non-draggable (Chrome, Safari)
+  $img.css('-webkit-user-drag', 'none');
+
+  // Firefox, IE<=10
+  // First remove existing handler so we don't keep adding handlers.
+  $img.off('dragstart.image_output');
+  $img.on('dragstart.image_output', function() { return false; });
+
+  // Disable selection of image and text when dragging in IE<=10
+  $el.on('selectstart.image_output', function() { return false; });
```

Comment:
Do you also need to remove this handler first, the way you did for `dragstart.image_output`?

Reply:
Yup, I was being sloppy with this part :(